doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-coverage-report-dmn-based-proc.adoc

```diff
@@ -0,0 +1,13 @@
+[id='test-scenarios-coverage-report-dmn-based-proc']
+= Coverage report for DMN-based test scenarios
+
+Follow the procedure below to generate a coverage report for DMN-based test scenarios.
+
+.Prerequisites
+* The DMN-based test scenario template has been created for the selected test scenario.For more information about creating DMN-based test scenarios, see xref:test-designer-create-test-template-dmn-based-proc[].
+* The individual test scenarios have been defined. For more information about defining a test scenario, see xref:test-designer-test-scenario-definition-proc[].
+
+.Procedure
```

Comment:
Question - Should there be a step or steps to help the user navigate to the test scenarios designer? I see most test scenario related docs do not have this.

Something like this example:
* Log in to {CENTRAL}.
* Click *Menu* -> *Design* -> *Projects*, then *{PROJECT}*.

Reply:
There are steps to help the user to the test scenarios designer. It is already defined in section 3.4 of the test scenario document.